doc/tutorials/cascading.md

```diff
@@ -0,0 +1,131 @@
+## Order of namespaces
+
+This tutorial assumes that you know what namespaces are. You might want to read
+the [Namespaces tutorial](/doc/tutorials/namespaces.md) first.
+
+In Elektra, the default order of namespaces is as follows:
+
+ * [spec](https://github.com/ElektraInitiative/libelektra/blob/master/doc/help/elektra-namespaces.md#spec) (modify elektra lookup behaviour)
+ * [proc](https://github.com/ElektraInitiative/libelektra/blob/master/doc/help/elektra-namespaces.md#proc) (process-related information)
+ * [dir](https://github.com/ElektraInitiative/libelektra/blob/master/doc/help/elektra-namespaces.md#dir) (directory-related information, e.g. `.git` or `.htaccess`)
+ * [user](https://github.com/ElektraInitiative/libelektra/blob/master/doc/help/elektra-namespaces.md#user) (user configuration)
+ * [system](https://github.com/ElektraInitiative/libelektra/blob/master/doc/help/elektra-namespaces.md#system) (system configuration)
+ * [cascading](https://github.com/ElektraInitiative/libelektra/blob/master/doc/help/elektra-namespaces.md#cascading) (keys without a namespace)
+
+Looking at this order, we can see that if a configuration option isn't specified
+by the user (then it would be in the `user` namespace), it will be loaded from
+the `system` namespace. In this case, the option in the `system` namespace works
+like a default value.
```

Comment:
`default value` is a rather vague term which does not help here. On cascading lookup `user` is preferred to `system`, not more, not less.